src/components/Task/TaskRow/TaskRow.test.tsx

```diff
@@ -179,7 +179,7 @@ describe('TaskRow', () => {
         },
       });
 
-      const { findByText, getByRole } = render(
+      const { getByText, getByRole } = render(
```

Comment:
Nice work making this synchronous! We can remove the `async` from the test function now.